### DISCUSSION TITLE
Make toggle terminal remember it's size

### DISCUSSION THREAD

**Discussion by DeNice-r** [NvChad/NvChad#2918] [Q&A] [answered]
I am new to nvchad and nvim. I was trying to learn and customize on the fly, and stumbled upon an unexpected (from my pov) behavior: when I open a toggleable (`<A-v>`) terminal, change it's size and close+reopen (`<A-v>` twice) it, it resets to the original size.

Is there a way to make it keep the adjusted size?

**Comment by siduck** [comment, marked as answer]
thats gonna be tough, cuz the size is only taken from your function args or term config in chadrc. 

I'd have to check via winresized event and do lot of modifications to the code to the point that the term module wont be minimal anymore :fearful:

**Reply by DeNice-r** [reply]
I appreciate the insights! I have dug through the source and came to a similar conclusion, so I settled with setting default width/height of the terminals to the value I like.

Thanks for your work!

**Reply by siduck** [reply]
@DeNice-r  if you dont want the horizontal terminal mess up your layout then try the floating terminal. You can do some math and position it bottom horizontal ,  our cheatsheet window is floating even tho it looks full-screen 

![image](https://github.com/NvChad/NvChad/assets/59060246/bba060eb-f54b-45d7-8d05-319330a8fb3d)

**Reply by DeNice-r** [reply]
Will do, great suggestion, thanks.
